packages/terra-icon/src/svg/volumeSet_decrease.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 48 48"><path d="M29.121 48.059L15.087 34.024H.05V13.976h15.037L29.121-.059zm6.323-35.406l-2.131 2.131a13.009 13.009 0 010 18.43l2.131 2.131a16.039 16.039 0 000-22.692z" data-name="256"/></svg>
```

Comment:
Please add a newline to the end of all files.

Reply:
Same comment about generated files. N/m: not worth it. Maybe we can fix the script to emit newline. But low value if these are generated.

Reply:
Fixed the migration script to inject new lines.